fe/fe-core/src/test/java/com/starrocks/analysis/SelectStmtWithDecimalTypesNewPlannerTest.java

```diff
@@ -65,7 +65,7 @@ public void testNullif() throws Exception {
         String thrift = UtFrameUtils.getPlanThriftStringForNewPlanner(ctx, sql);
         Assert.assertTrue(thrift.contains(expectString));
 
-        thrift = UtFrameUtils.getPlanThriftString(ctx, sql);
+        thrift = UtFrameUtils.getPlanThriftStringForNewPlanner(ctx, sql);
```

Comment:
Why need to change it to new planner?

Reply:
There is a problem with the be startup of FE-UT. By the way, migrate the old UT to new planner